i18n/en.json

```diff
@@ -0,0 +1,241 @@
+        ],
+        "inputOutput": [
+          "Looks like Abitikku is not able to write to this location of the drive.",
+          "This error is usually caused by a faulty drive, reader, or port.",
+          "\n\nPlease try again with another drive, reader, or port."
+        ],
+        "childWriterDied": [
+          "The writer process ended unexpectedly.",
+          "Please try again, and contact the development team if the problem persists."
+        ],
+        "unsupportedProtocol": "Only http:// and https:// URLs are supported."
+      }
+    },
+    "errors": {
+      "EPERM": {
+        "title": "You're not authorized to perform this operation",
+        "description": "Please ensure you have necessary permissions for this task"
+      },
+      "EACCES": {
+        "title": "You don't have access to this resource",
+        "description": "Please ensure you have necessary permissions to access this resource"
+      },
+      "ENOMEM": {
+        "title": "Your system ran out of memory",
+        "description": "Please make sure your system has enough available memory for this task"
+      },
+      "errorCode": "Error code: {{code}}",
+      "genericError": "An error occurred"
+    },
+    "sudo": {
+      "privilegedAccessRequired": "Abitikku needs privileged access in order to flash disks.\n\nType your password to allow this."
+    }
+  },
+  "gui": {
+    "store": {
+      "missingDrive": "Missing drive",
+      "missingDrives": "Missing drives",
+      "invalidDrive": "Invalid drive: {{drive}}",
+      "invalidDrives": "Invalid drives: {{drives}}",
+      "driveUnavailable": "The drive is not available: {{drive}}",
+      "driveWriteProtected": "The drive is write-protected",
+      "driveTooSmall": "The drive is not large enough"
+    },
+    "menu": {
+      "mainLabel": "View",
+      "toggleDevTools": "Toggle Developer Tools",
+      "reportIssue": "Report an issue",
+      "about": "About Abitikku"
+    },
+    "mainPage": {
+      "driveTitle": {
+        "noDescription": "Untitled Device",
+        "noTargets": "No targets found"
+      }
+    },
+    "flash": {
+      "success": "Flash complete!",
+      "failure": "Oops! Looks like the flash failed.",
+      "eta": "ETA: {{seconds}}",
+      "errorModal": {
+        "title": "Attention",
+        "action": "Retry"
+      }
+    },
+    "dialog": {
+      "imageSelect": {
+        "osImages": "OS Images",
+        "all": "All"
+      },
+      "warningTitle": "Attention"
+    },
+    "progress-status": {
+      "starting": "Starting...",
+      "decompressing": "Decompressing...",
+      "flashing": "Flashing...",
+      "finishing": "Finishing...",
+      "validating": "Validating...",
+      "failed": "Failed"
+    },
+    "image-writer": {
+      "processEnded": {
+        "title": "The writer process ended unexpectedly",
+        "description": "Please try again, and contact the development team if the problem persists"
```

Comment:
`development team` -> `developer team`?

Reply:
In my opinion the terms work like this:

`Development team`: Team that develops the application
`Developer team`: A team that consists of developers

Thus `Development team` fits better.